cw/cw305/capture.py

```diff
@@ -26,9 +28,30 @@
 # Shared options for "capture aes" and "capture sha3".
 opt_num_traces = typer.Option(None, help="Number of traces to capture.")
 opt_plot_traces = typer.Option(None, help="Number of traces to plot.")
+opt_scope_type = typer.Option("cw", help=("cw or waverunner as a scope for "
+                                          "batch capture"))
```

Comment:
nit: enums are safer:

```suggestion
from enum import Enum

...

class ScopeType(str, Enum):
    cw = "cw"
    waverunner = "waverunner"

opt_scope_type = typer.Option(ScopeType.cw, help=("Scope type"))
```

Please take a look at [here](https://typer.tiangolo.com/tutorial/parameter-types/enum/) for more details.